mitodl/mitxpro
# asadiqbal08/mitxpro-1215 - Management Command to revoke Program/Run Certificates.

#### Pre-Flight checklist


#### What are the relevant tickets?
#1215 

#### What's this PR do?
As a member of the course team, I need a safe and reliable way to revoke a certificate.

#### How should this be manually tested?

- Generate some certificates for `CourseRunCertificates` and `ProgramCertificates` models. 
- Be-default, all certificates should not be marked `revoked`
- Execute `./manage.py revoke_certificate --user={YOUR-USER} --program={PROGRAM_READABLE_ID}` in order to revoke a ProgramCertificate
    - Corresponding `CourseRunCertificate` should also be marked `revoked/unrevoked` that are associated with that program and user if `--include-program-courses` option is passed along with.
 
- Execute `./manage.py revoke_certificate --user={YOUR-USER} --run={COURSEWARE_ID}` in order to revoke a CourseRunCertificate
- User should not be able to see a `View Certificate` button on dashboard for  **revoked** course/program certificates. 
- Try to execute the command with different random parameters. 
- Changes should be rendered over Django admin pages.
